src/modals/masq_favorite_modal.js

```diff
@@ -40,10 +40,11 @@ export default class MasqFavoriteModal {
   }
 
   async openMasq () {
-    await store.login()
-    if (await store.isLoggedIn()) {
-      this.close()
+    try {
+      await store.login()
+    } catch(e) {
```

Comment:
could we at least log a message in these cases, to prevent errors from being unnoticed ?

Reply:
yes you're right ! :+1: